editor/project_manager.cpp

```diff
@@ -2684,8 +2684,26 @@ ProjectManager::ProjectManager() {
 
 	_load_recent_projects();
 
-	if (EditorSettings::get_singleton()->get("filesystem/directories/autoscan_project_path")) {
-		_scan_begin(EditorSettings::get_singleton()->get("filesystem/directories/autoscan_project_path"));
+	DirAccessRef dir_access = DirAccess::create(DirAccess::AccessType::ACCESS_FILESYSTEM);
+
+	Variant default_project_path = EditorSettings::get_singleton()->get("filesystem/directories/default_project_path");
```

Comment:
```suggestion
	String default_project_path = EditorSettings::get_singleton()->get("filesystem/directories/default_project_path");
```

And then you don't need to cast to `String` further down. Same for autoscan.